ferrocene/tools/self-test/src/error.rs

```diff
@@ -113,6 +115,7 @@ impl Error {
             Error::DuplicateTargetLibrary { .. } => 9,
             Error::TargetLibraryDiscoveryFailed { .. } => 10,
             Error::CCompilerNotFound { .. } => 11,
+            Error::CliArgsProvided => 12,
```

Comment:
We also need to list the error code in https://public-docs.ferrocene.dev/main/user-manual/self-test/error-codes.html.

Reply:
https://github.com/ferrocene/ferrocene/pull/907/commits/f9ba66227d85d938e28680641c81f2aadd915e3b